### PR TITLE
Added some ignore pattern for import/no-extraneous-dependencies

### PR DESCRIPTION
**Pull request by chentsulin**
Close https://github.com/airbnb/javascript/issues/1168

### CODE REVIEW

File: packages/eslint-config-airbnb-base/rules/imports.js [hunk 1]
```diff
@@ -68,7 +68,17 @@ module.exports = {
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md
     // paths are treated both as absolute paths, and relative to process.cwd()
     'import/no-extraneous-dependencies': ['error', {
-      devDependencies: ['spec/**', 'test/**', 'tests/**', '**/__tests__/**'],
+      devDependencies: [
+        'spec/**', 
+        'test/**', 
+        'tests/**', 
+        '**/__tests__/**',
+        '**/webpack.config.js',
```
Comment (ljharb) [inline packages/eslint-config-airbnb-base/rules/imports.js]: why would anyone have their webpack config not at the root? the leading "**/" should be removed - same for all the other config files.

Comment (chentsulin) [inline packages/eslint-config-airbnb-base/rules/imports.js]: I ever see some usage case like:

`examples/webpack.config.js` 

or 

```
webpack/webpack.config.js
webpack/webpack.config.dev.js
webpack/webpack.config.prod.js
webpack/webpack.config.test.js
```

But I can take those changes if it is too much.

Comment (ljharb) [inline packages/eslint-config-airbnb-base/rules/imports.js]: hmm - i guess that makes sense. let's keep them for now.

File: packages/eslint-config-airbnb-base/rules/imports.js [hunk 2]
```diff
@@ -68,7 +68,17 @@ module.exports = {
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md
     // paths are treated both as absolute paths, and relative to process.cwd()
     'import/no-extraneous-dependencies': ['error', {
-      devDependencies: ['spec/**', 'test/**', 'tests/**', '**/__tests__/**'],
+      devDependencies: [
+        'spec/**',
+        'test/**',
+        'tests/**',
+        '**/__tests__/**',
```
Comment (stevemao) [inline packages/eslint-config-airbnb-base/rules/imports.js]: Would be great to include `*.spec.js` as well :)

**Pull request merged**